src/EditorFeatures/Test2/Simplification/CastSimplificationTests.vb

```diff
@@ -7687,7 +7687,7 @@ End Class
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.Simplification)>
         <WorkItem(995855)>
-        Public Sub VisualBasic_Remove_UnnecessaryCastInTernaryExpression()
+        Public Sub VisualBasic_DontRemove_NecessaryCastInTernaryExpression1()
```

Comment:
Note: This test was simply wrong. Removing either cast to CByte(0) would result in the type of the ternary conditional changing from `Byte` to `Int32`. These should only be marked as redundant if the type of the ternary conditional is already `Int32`.